content/en/docs/concepts/deployment-models/index.md

```diff
@@ -0,0 +1,421 @@
+
+- The network consists of a number of connected subnets without any
+  overlap in IP or VIP ranges.
+- All {{< gloss >}}workload instance{{< /gloss >}}s can reach each other directly
+  without an Istio gateway.
+
+A single network allows Istio to configure service consumers in a uniform
+way across the mesh with the ability to directly address workload instances.
+
+{{< image width="50%"
+    link="./single-net.svg"
+    alt="A service mesh with a single network"
+    title="Single network"
+    caption="A service mesh with a single network"
+    >}}
+
+### Multiple networks
+
+You can span a single service mesh across multiple networks; such a
+configuration is known as **multi-network**.
+
+Multiple networks afford the following capabilities beyond that of single networks:
+
+- An existing network topology
+- Overlapping IP or VIP ranges for **service endpoints**
+- Crossing of administrative boundaries
+- Fault tolerance
+- Scaling of network addresses
+
+In this model, the workload instances in different networks can only reach each
+other through one or more [Istio gateways](/docs/concepts/traffic-management/#gateways).
+Istio uses **partitioned service discovery** to provide consumers a different
+view of {{< gloss >}}service endpoint{{< /gloss >}}s. The view depends on the
+network of the consumers.
+
+{{< image width="50%"
+    link="./multi-net.svg"
+    alt="A service mesh with multiple networks"
+    title="Multi-network deployment"
+    caption="A service mesh with multiple networks"
+    >}}
+
+## Mesh models
+
+Istio supports two mesh models:
+
+- Single mesh
+- Multiple meshes, also known as **multi-mesh**
+
+### Single mesh
+
+The simplest Istio deployment is a single mesh. Within a mesh, service names are
+unique. For example, only one service can have the name `foo/mysvc`.
+Additionally, workload instances all share a common identity and authentication
+infrastructure and can authenticate each other. With a single mesh you can do
+the following tasks:
+
+- Span [one or more clusters](#cluster-models)
+- Span [one or more networks](#network-models)
+- Use [namespaces for tenancy](#namespace-tenancy)
+
+### Multiple Meshes
+
+You can deploy multiple meshes; such a configuration is known as
+**multi-mesh**.
+
+Multiple meshes afford the following capabilities beyond that of a single mesh:
+
+- Organizational boundaries: lines of business
+- Service name or namespace collisions: multiple distinct uses of the `default`
+  namespace
+- Stronger isolation: isolating test workloads from production workloads
+
+In the most basic multiple mesh model, each mesh forms a strict
+boundary and there is no communication between meshes. However, you can enable
+inter-mesh communication with **mesh peering**. When
+peering, each mesh can export a set of services and identities, which all
+participating meshes can recognize.
+
+{{< image width="50%"
+    link="./multi-mesh.svg"
```

Comment:
Should we show only certain services being exported? It is an important part of mesh federation that by default nothing is exposed, each service must explicitly choose to export itself to another mesh.

Reply:
I also had concerns showing all the services connecting with each other. That's why I added the trust bundles to the diagram. 
@smawson Since the default is to have nothing exposed, I would suggest I change the diagram to only show a connection between `Service A` and `Service B` and a connection between `Service C` and `Service D`.
Would that change address your concerns?

Reply:
I was thinking more like A or B can talk to C but they can't talk to D.

Reply:
Yeah should be an easy fix .. we already do something similar in another diagram.